server/questions.js

```diff
@@ -121,50 +131,17 @@ var updateQuestionByType = function(qId, infoToUpdate, callback){
 		if(err){
```

Comment:
change this function to updateQuestionById instead of type

Reply:
done